drivers/shared/capabilities/defaults.go

```diff
@@ -36,7 +36,6 @@ func NomadDefaults() *Set {
 // https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
 func DockerDefaults() *Set {
 	defaults := NomadDefaults()
-	defaults.Add("NET_RAW")
 	return defaults
```

Comment:
Just wanting to confirm this removal is necessary and appropriate. I thought this was in place specifically because on Linux Docker grants NET_RAW which is problematic and Nomad explicitly removes it. So I'm not sure if removing it from here is intended.

Reply:
@gulducat any code change we're making in this PR should get pulled out into its own PR so those details can be hammered out (and so that we're not comingling commits of OSS production code with internal tooling code).

Reply:
yep, agreed.  I vaguely recall before I was out on vacay/illness that we found this to be unnecessary (with windows 2019 instead of 2022 iirc), so hopefully we can just drop it, but otherwise yes we'll move any functional changes like this to separate PR(s).

Reply:
I was mistaken, we can't do nothing, but I can move the change into the test harness instead: [e677ac2](https://github.com/hashicorp/nomad/pull/17103/commits/e677ac2b90ef5f5ff2c721dc5615758e49996069) -- any objections to that until we get around to addressing the real issue?